packages/fxa-content-server/app/scripts/templates/settings.mustache

```diff
@@ -5,7 +5,10 @@
     </div>
   {{/ccExpired}}
   <header id="fxa-settings-header">
-    <h1 id="fxa-manage-account"><span class="fxa-account-title">{{#t}}Firefox Accounts{{/t}}</span></h1>
+    <div class="header-lockup">
+      <h1 id="fxa-manage-account"><span class="fxa-account-title">{{#t}}Firefox Accounts{{/t}}</span></h1>
+      <a href="/beta/settings" class="nu-settings-button">Switch to new design</a>
```

Comment:
clicking this doesn't actually navigate me to new settings... ???

Reply:
I can't seem to figure out why this isn't work 🤔 I'm going to keep poking at it, but maybe I need to add a route handler? or add the hostname into the link.